### ISSUE TITLE
Build 6824762 fails

### ISSUE THREAD

**Issue by justb4** [rtfd/readthedocs.org#3709]
## Details

* Project URL: http://smartplatform.readthedocs.io/en/latest/
* Build URL (if applicable): https://readthedocs.org/projects/smartplatform/builds/6824762/
* Read the Docs username (if applicable): geonovum

## Expected Result

Document built on GitHub commit/push.

## Actual Result

Build failures. Tried multiple times and have reset build env. Text:

```
Build #6824762
latest
Build failed
Having trouble with your build environment? Try resetting it.

Error
There was a problem with Read the Docs while building your documentation. 
Please report this to us with your build id (6824762).

git clone --recursive https://github.com/Geonovum/smartemission.git .
git show-ref remotes/origin/master
git checkout --force origin/master
```

**Comment by RichardLitt** [comment]
I believe this is part of a few errors caused on our side. Fixes are being worked on now: see #3707.

**Comment by justb4** [comment]
Doc is building now. Closing this one. Keep up the great work!
